src/index.ts

```diff
@@ -296,6 +300,9 @@ const vimPlugin = ViewPlugin.fromClass(
           var vimPlugin = cm.state.vimPlugin;
 
           if (vim && !vim.insertMode && !cm.curOp?.isVimOp) {
+            if (text == "\0\0") {
```

Comment:
we should use triple equals here
```suggestion
            if (text === "\0\0") {
```